server/routers/errorRouter.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const errorController = require('../controllers/errorController');
```

Comment:
See https://github.com/oslabs-beta/Kubiquity/pull/12/files#r664691786 (and relevant for the rest of this file)